install.html

```diff
@@ -74,11 +74,6 @@
                     <p>This page helps you to choose and install a hypervisor into your personal computer.</p>
                 </article>
 
-                <details class="showme">
-                    <summary>Show me what it looks like...</summary>
-                    and this is the bulk...  blah blah
-                </details>
-
```

Comment:
This is here waiting for a few style changes and then to be taken into use elsewhere. Won't be removed for now.

Reply:
Ok! Added this back.